EIPS/eip-3403.md

```diff
@@ -0,0 +1,69 @@
+---
+eip: 3403
+title: Partial removal of refunds
+author: Vitalik Buterin (@vbuterin), Martin Swende (@holiman)
+discussions-to: https://ethereum-magicians.org/t/eip-3298-removal-of-refunds/5430
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-16
+---
+
+## Simple Summary
+
+Remove gas refunds for SELFDESTRUCT, and restrict gas refunds for SSTORE to one specific case.
+
+## Motivation
+
+Gas refunds for SSTORE and SELFDESTRUCT were originally introduced to motivate application developers to write applications that practice "good state hygiene", clearing storage slots and contracts that are no longer needed. However, they are not widely used for this, and poor state hygiene continues to be the norm. It is now widely accepted that the only solution to state growth is some form of [statelessness or state expiry](https://hackmd.io/@HWeNw8hNRimMm2m2GH56Cw/state_size_management), and if such a solution is implemented, then disused storage slots and contracts would start to be ignored automatically.
```

Comment:
```suggestion
Gas refunds for SSTORE and SELFDESTRUCT were originally introduced to motivate application developers to write applications that practice "good state hygiene", clearing storage slots and contracts that are no longer needed. However, they are not widely used for this, and poor state hygiene continues to be the norm. It is now widely accepted that the only solution to state growth is some form of statelessness or state expiry, and if such a solution is implemented, then disused storage slots and contracts would start to be ignored automatically.
```
External links are generally not allowed because they tend to 404 over time.  Recommend including references like this in the discussions-to thread.